glide.lock

```diff
@@ -36,16 +19,12 @@ imports:
   version: d6574a5bb1226678d7010325fb6c985db20ee458
   subpackages:
   - coordinate
-- name: github.com/inconshreveable/mousetrap
-  version: 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
```

Comment:
@AlekSi I'm curious what you told glide so it ended up without this dependency?

Reply:
Cobra [uses it](https://github.com/spf13/cobra/blob/master/command_win.go), Glide adds deps for all common build tags.